client/src/components/login/forms.js

```diff
@@ -12,24 +11,25 @@ const renderTextField = ({input, label, meta: {touched, error}, ...custom}) => (
     underlineStyle={{borderColor: '#222'}}
     underlineFocusStyle={{borderColor: '#222'}}
     errorText={touched && error}
+    className='login-form-field'
     {...input}
     {...custom}
     />
 )
 
 const loginForm = props => {
   const { handleSubmit } = props
-  
+
   return (
     <form onSubmit={handleSubmit} name='login-form'>
       <div>
-        <Field name="username" component={renderTextField} type="text" placeholder="username"/>
+        <Field name='username' component={renderTextField} type='text' placeholder='username' />
       </div>
       <div>
-        <Field name="password" component={renderTextField} type="password" placeholder="password"/>
+        <Field name='password' component={renderTextField} type='password' placeholder='password' />
       </div>
-      <button className='signin-button' type="submit">login</button>
-    </form> 
+      <button className='signin-button' type='submit'>login</button>
```

Comment:
as you can tell, my linter isdn't on lol (sorry)

Reply:
lol it's fine i just linted whatever files i was in but didn't want to lint everything and end up with a huge changeset--it doesn't really matter :)